README.md

```diff
@@ -1,4 +1,4 @@
-# Dyno [![Build Status](https://secure.travis-ci.org/Netflix/dyno.png)](http://travis-ci.org/Netflix/dyno)
+# Dyno
```

Comment:
Yeah.. working on getting that back in, our guy doing the in house builds happened to grab the branch I was pull requesting from.